src/App.jsx

```diff
@@ -14,33 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const ALPHABET_SORT = 'alphabetically';
+const LENGTH_SORT = 'byLength';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getSorted = (goods, sortField, isReversed) => {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  preparedGoods = preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case ALPHABET_SORT:
+        return good1.localeCompare(good2);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case LENGTH_SORT:
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+};
```

Comment:
Move it to separate file to make this component more compact and readable

Reply:
it's readable enough

Reply:
> it's readable enough, what exactly you can't read? what exactly you don't understand?

In this case it's ok. But it's a good idea to separate logic, you can create utils/helpers file/folder and keep in them